<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav1)"/>
  <arg name="debug" default="false" />

  <arg unless="$(arg debug)" name="launch_prefix" value=""/>
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb -ex run --args"/>

  <group ns="$(arg uav_name)">

    <node pkg="nodelet" type="nodelet" name="balloon_planner" args="standalone balloon_planner/BalloonPlanner" output="screen" launch-prefix="$(arg launch_prefix)" >

      <param name="uav_name" type="string" value="$(arg uav_name)" />
      <!-- <rosparam file="$(find balloon_planner)/config/simulation.yaml" command="load" /> -->

      <!-- Subscribers -->
      <remap from="~cloud_balloon_in" to="balloon_localize/cloud" />

      <!-- Publishers -->
      <remap from="~odom_balloon_out" to="odom_balloon_out" />

    </node>

  </group>

</launch>
